torchvision/csrc/io/image/cpu/decode_png.cpp

```diff
@@ -23,6 +23,7 @@ torch::Tensor decode_png(
     const torch::Tensor& data,
     ImageReadMode mode,
     bool allow_16_bits) {
+  C10_LOG_API_USAGE_ONCE("torchvision.io.decode_png");
```

Comment:
Same here and in all other places where the string on the C++ side matches the one on Python.